xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -862,5 +860,19 @@ namespace XBMCAddon
       }
       return els;
     }
+
+    CVideoInfoTag* ListItem::GetVideoInfoTag()
+    {
+      // make sure the playcount is reset to -1
+      if (!item->HasVideoInfoTag())
+        item->GetVideoInfoTag()->ResetPlayCount();
+
+      return item->GetVideoInfoTag();
+    }
+
+    const CVideoInfoTag* ListItem::GetVideoInfoTag() const
+    {
+      return item->GetVideoInfoTag();
```

Comment:
No this is the `const` version so we can't call into the non-`const` version. `CFileItem::GetVideoInfoTag() const` simply returns `m_videoInfoTag` independent of whether it's `nullptr` or not.